packages/office-ui-fabric-react/src/components/Dialog/DialogContent.styles.ts

```diff
@@ -1,18 +1,141 @@
-import { IDialogContentStyleProps, IDialogContentStyles } from './DialogContent.types';
+import {
+  IDialogContentStyleProps,
+  IDialogContentStyles,
+  DialogType
+} from './DialogContent.types';
+import {
+  IStyle,
+  ITheme,
+  FontWeights,
+} from '../../Styling';
 
 export const getStyles = (
   props: IDialogContentStyleProps
 ): IDialogContentStyles => {
+  const {
+    className,
+    theme,
+    isLargeHeader,
+    isClose,
+    hidden,
+    isMultiline,
+  } = props;
+
+  const { palette, fonts } = theme;
+  const dialogLgHeaderBackgroundColor = palette.themePrimary;
```

Comment:
dialogLgHeaderBackgroundColor is redundant as its only used once.  just use `palette.themePrimary` where it's used in the styles and nix the extra variable.